### PR TITLE
Remove phobos dependency from several tests

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
The test suite should not depend on phobos (and only refer to druntime if necessary)

### CODE REVIEW

File: test/runnable/test15.d [hunk 1]
```diff
@@ -280,10 +277,827 @@ void test20()
 
 void test21()
 {
+    static immutable string[] lines = [
```
Comment (Geod24) [inline test/runnable/test15.d:280]: Just use an import expression ?

**Comment by Geod24** [comment]
Nice! I think we can just use an import exp for the last one (and remove most of the diff), instead of making the test case unreadable.

File: test/runnable/test15.d (hunk 1 above)
Comment (MoonlightSentinel) [reply test/runnable/test15.d]: Done

**Review by ibuclaw — APPROVED** [review]

**Review by WalterBright — COMMENTED** [review]
good!

**Pull request merged**